layout/_partial/post/actions.ejs

```diff
@@ -38,13 +38,19 @@
                 </a>
             </li>
         <% } %>
-        <% if (post.comments && (theme.disqus_shortname || theme.duoshuo_shortname)) { %>
-            <li class="post-action">
-                <a class="post-action-btn btn btn--default" href="#<%= (theme.duoshuo_shortname ? 'ds-thread' : 'disqus_thread') %>">
+        <% if (post.comments) { %>
+	    <li class="post-action">
+              <% if (theme.gitment.enable) { %>
+                <a class="post-action-btn btn btn--default" href="#gcomment">
+                     <i class="fa fa-comment-o"></i>
+                </a>
+              <% } else { %>
```

Comment:
you need to check `else if (theme.disqus_shortname || theme.duoshuo_shortname)` otherwise it will display empty elements.

Reply:
Another careless action... omg...